### ISSUE TITLE
Keep audio playing on the device when recording to file

### ISSUE THREAD

**Issue by UtaUtaUtau** [Genymobile/scrcpy#3821]
Hopefully I didn't miss anything in the audio readme but if I remember correctly, sndcpy can keep the audio playing on the phone. Would it possible to have that here as well? Maybe only available when recording to a file as I don't really know if it would make sense to keep audio as it casts the video to the window.

Amazing work as always though!

**Comment by rom1v** [comment]
> sndcpy can keep the audio playing on the phone

Correct. It could not capture the "remote submix" source (it used a `MediaProjection` from an Android app), and as a consequence the audio kept playing on the device.

As soon as we capture the "remote submix" source with an `AudioRecord`, the audio is forwarded and disabled on the device. I am not aware of any way to change this behavior.

**Comment by manbug10** [comment]
Does it happen to anyone that when playing and activating the microphone, the game audio is deactivated and when the microphone is turned off, the audio returns?

**Comment by namehehe** [comment]
> Does it happen to anyone that when playing and activating the microphone, the game audio is deactivated and when the microphone is turned off, the audio returns?

I encountered the same problem. how to solve
